### PR TITLE
Update to README and logo

### PR DESCRIPTION
**Pull request by JWBurgers**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,79 @@
...
+
+We automatically on-board users to Lightning through the integration of our lightning service provider (LSP) [Blocktank](https://github.com/synonymdev/blocktank-client). Developers can also create their own LSP using Blocktank. In the future, we will support users to connect to their own Lightning and Bitcoin nodes,  select different types of LSPs, couple hardware wallets, and manage their own channels.  
+
+Users can back up all their on-chain Bitcoin data with a standard BIP-39 seed. Lightning channel data should automatically be stored on a backup server. The data needs to be encrypted on the client-side before it is sent to a server to ensure privacy. You can view our code [here](https://github.com/synonymdev/bitkit-backup-client). 
+
+The main other features Bitkit supports are the creation of social profiles; the importing and managing of contacts; dynamic payment profiles; key-based account logins; and the display of public data feeds and private account data. You can experiment with these features in our [playground](https://synonym.to/products/slashtags#playground).
```
Comment (coreyphillips) [inline README.md:37]: Suggested Change: Change "The main other features" to "The other main features"

File: README.md [hunk 2]
```diff
@@ -1,32 +1,79 @@
...
+
+While much of the power of Slashtags comes via these hyperdrives, the keypairs that can be generated with Slashtags also can have important utility on their own. Bi-lateral, peer to peer authentication between a user and a server can, for example, be realized just on the basis of keypairs. 
+
+All Slashtags data is exchanged within a Kademlia-based distributed hash table system.  
+
+You can read more about Slashtags on our [website](https://synonym.to/products/slashtags) and on our SDK's [Github page](https://github.com/synonymdev/slashtags). Slashtags predominantly builds on the lower-level functionality of the Hypercore stack. You can learn more about the Hypercore stack [here](https://hypercore-protocol.org/) as well as from the two main Github projects [here](https://github.com/hypercore-protocol) and [here](https://github.com/hyperswarm). 
+
+Slashtags user data should be automatically on a [seeder server](https://github.com/synonymdev/slashtag-seeding-server). In this way, user data is always available. In addition, it allows the user to restore all their Slashtags data from their BIP-39 seed.
```
Comment (coreyphillips) [inline README.md:49]: Suggested Change:
Change "user data should be automatically on a " to "user data is automatically added to a ".

File: README.md [hunk 3]
```diff
@@ -1,32 +1,79 @@
...
+
+All Slashtags data is exchanged within a Kademlia-based distributed hash table system.  
+
+You can read more about Slashtags on our [website](https://synonym.to/products/slashtags) and on our SDK's [Github page](https://github.com/synonymdev/slashtags). Slashtags predominantly builds on the lower-level functionality of the Hypercore stack. You can learn more about the Hypercore stack [here](https://hypercore-protocol.org/) as well as from the two main Github projects [here](https://github.com/hypercore-protocol) and [here](https://github.com/hyperswarm). 
+
+Slashtags user data should be automatically on a [seeder server](https://github.com/synonymdev/slashtag-seeding-server). In this way, user data is always available. In addition, it allows the user to restore all their Slashtags data from their BIP-39 seed. 
+
+
+## Support
+If you are experiencing any problems with Bitkit, please open an issue and use the template provided.
```
Comment (coreyphillips) [inline README.md]: Suggested Change:
Change "please open an issue and use the template provided. " to "please open an issue using the template provided or reach out to us on [Telegram](https://t.me/bitkitchat). ".

File: README.md [hunk 4]
```diff
@@ -1,32 +1,79 @@
...
+---
 
-1. Clone Bitkit:
+## Overview
+
+Bitkit enables users to reclaim their digital life.
+
+Bitkit puts users in control of their money and payments, social profile, contacts, and payment preferences; allows users to authenticate to servers with just a tap; and allows users to stream data from public feeds and your accounts, all within one application. What Bitcoin has done for money and payments, Bitkit also extends to other areas of digital life. 
+
+Bitkit is a mobile application available on Android and iOS, and implemented using React-native.
```
Comment (pwltr) [inline README.md]:
```suggestion
Bitkit is a mobile application available on Android and iOS, and implemented using React Native. 
```

File: README.md [hunk 5]
```diff
@@ -1,32 +1,79 @@
-# Bitkit - Mobile
+<p align="center">
+  <a href="https://github.com/synonymdev/bitkit" title="Bitkit">
+    <img alt="bitkit" src="./src/assets/bitkit_logo_readme.png" width="150"></img>
+  </a>
+</p>
 
-Bitkit is a self-custodial mobile keychain and wallet application that supports Bitcoin, Lightning Network and Slashtags.
+<h3 align="center">Bitkit Mobile</h3>
```
Comment (pwltr) [inline README.md:7]:
```suggestion
<h3 align="center">Bitkit</h3>
```

File: README.md [hunk 6]
```diff
@@ -1,32 +1,79 @@
...
 
-Bitkit includes backup and restore features utilizing [Backpack](https://github.com/synonymdev/backpack-client), and supports [Blocktank](https://github.com/synonymdev/blocktank-client) LSP services.
+---
 
-[![Test status](https://github.com/synonymdev/bitkit/workflows/tests/badge.svg)](https://github.com/synonymdev/bitkit/actions)
+**⚠️ Bitkit mobile is still in beta. Please use at your own risk.⚠️**
```
Comment (pwltr) [inline README.md:11]:
```suggestion
**⚠️ Bitkit is still in beta. Please use at your own risk.⚠️**
```

File: README.md [hunk 7]
```diff
@@ -1,32 +1,79 @@
...
+
+
+## The problem
+
+Society, the economy, and the Web have been overcome by oppressive central controls, resulting in censorship, privacy breaches, and monopoly behaviors that stifle the marketplace of goods and ideas. Users need P2P tools that set them free.   
+
+
+## Bitkit's solution
+
+Bitkit offers a Bitcoin wallet that supports both on-chain and lightning transactions. The application runs a custom, non-routing Lightning node created with the [Lightning Development Kit] (https://github.com/lightningdevkit) on the user's mobile phone. It requires a connection to an external server running a Bitcoin full node and an [Electrum server]( https://github.com/synonymdev/react-native-electrum-client).
```
Comment (pwltr) [inline README.md:31]:
```suggestion
Bitkit offers a Bitcoin wallet that supports both on-chain and Lightning transactions. The application runs a custom, non-routing Lightning node created with the [Lightning Development Kit] (https://github.com/lightningdevkit) on the user's mobile phone. It requires a connection to an external server running a Bitcoin full node and an Electrum server.
```

File: README.md (hunk 2 above)
Comment (pwltr) [inline README.md:49]:
```suggestion
Slashtags user data should be automatically on a [seeding server](https://github.com/synonymdev/slashtag-seeding-server). In this way, user data is always available. In addition, it allows the user to restore all their Slashtags data from their BIP-39 seed. 
```

File: README.md [hunk 8]
```diff
@@ -1,32 +1,79 @@
...
+
+The main other features Bitkit supports are the creation of social profiles; the importing and managing of contacts; dynamic payment profiles; key-based account logins; and the display of public data feeds and private account data. You can experiment with these features in our [playground](https://synonym.to/products/slashtags#playground).  
+
+All these additional features are powered by Slashtags: an open-source protocol for creating secure and scalable peer-to-peer applications. You can view our JavaScript-based software development kit which has been used for the Bitkit wallet [here](https://github.com/synonymdev/slashtags). 
+
+The Slashtags protocol allows for the creation of cryptographic keypairs, known as "slashtags", derived from the same BIP-39 seed as the user's Bitcoin wallet. Typically these slashtags are associated with networked “drives” known as hyperdrives, which can be discovered, read, and copied by peers on a network. The group of peers that stores some or all of the drive’s data is known as its swarm.
```
Comment (pwltr) [inline README.md:41]:
```suggestion
The Slashtags protocol allows for the creation of cryptographic keypairs, known as [slashtags](https://slashtags.to/), derived from the same BIP-39 seed as the user's Bitcoin wallet. Typically these slashtags are associated with networked “drives” known as hyperdrives, which can be discovered, read, and copied by peers on a network. The group of peers that stores some or all of the drive’s data is known as its swarm. 
```

Comment (JWBurgers) [reply README.md:41]: I link to this a few lines later.

File: README.md (hunk 1 above)
Comment (JWBurgers) [reply README.md:37]: It's fine

File: README.md (hunk 2 above)
Comment (JWBurgers) [reply README.md:49]: Writing from the perspective what a developer should do.

**Review by Jasonvdb — APPROVED** [review]

**Pull request merged**